GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/GameDataController.cs

```diff
@@ -0,0 +1,90 @@
+#define ONLINE
+//#define OFFLINE
```

Comment:
What's this? Is this a temporary solution before you add the Editor code?

Reply:
From what I research, the Editor code will change depending the mode it's execute in unity. The preprocessor/defines will change the code it will execute before compilation, so it will also work when the game is build into an apk.

Reply:
I get it. I assume we should flip this ONLINE and OFFLINE definition with Editor code but you're hard-coding it here. So is this a temporary solution? Will you add editor code later?